packages/ts/react-crud/src/autocrud.obj.css

```
@@ -1,4 +1,5 @@
 .auto-crud {
+  flex: 1;
```

Let's use `flex: auto`, it is more robust due to keeping flex-basis.

```suggestion
  flex: auto;
```